estate/models/estate_property.py

```diff
@@ -0,0 +1,78 @@
+from odoo import api, models, fields
+from dateutil.relativedelta import relativedelta
+
+
+def _default_date_availability():
+    return fields.Date.today() + relativedelta(months=3)
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Real Estate Property Information"
+
+    name = fields.Char(required=True)
+    description = fields.Text()
+    property_tags_ids = fields.Many2many("estate.property.tag")
```

Comment:
nitpick:
```suggestion
    property_tag_ids = fields.Many2many("estate.property.tag", string="Property Tags")
```
for x2m the convention is usually to keep the base name in the singular (since `_ids` is already plural)